easybuild/easyconfigs/e/expect/expect-5.45.4-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'ConfigureMake'
+
+name = 'expect'
+version = '5.45.4'
+
+homepage = 'https://core.tcl.tk/expect/index'
+description = """
+Expect is a tool for automating interactive applications 
+such as telnet, ftp, passwd, fsck, rlogin, tip, etc. 
+Expect really makes this stuff trivial. 
+Expect is also useful for testing these same applications. 
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+toolchainopts = {'pic': True}
+
+sources = ['%(name)s%(version)s.tar.gz']
+source_urls = [SOURCEFORGE_SOURCE]
+checksums = ['49a7da83b0bdd9f46d04a04deec19c7767bb9a323e40c4781f89caf760b92c34']
+
+patches = []
```

Comment:
@jose-d A small point, but if you don't have any patches it's fine to remove this parameter altogether.